### ISSUE TITLE
Perform accessibility review

### ISSUE THREAD

**Issue by jnahumphreys** [jnahumphreys/react-counter-app#54]
- Decrement button has no focusable styling
- Increment button has no focusable styling
- Reset button has no focusable styling
- Screen reader doesn't announce count area
- Screen reader doesn't announce change of count value
- Screen reader doesn't announce change in button states
- Screen reader doesn't announce button action area

**Comment by jnahumphreys** [comment]
Tickets created to improve accessibility: 

#57 
#58 

Will close.
